### DISCUSSION TITLE
AMO has no v2.21.0

### DISCUSSION THREAD

**Discussion by hellodword** [violentmonkey/violentmonkey#2182] [Q&A] [answered]
- https://addons.mozilla.org/en-US/firefox/addon/violentmonkey/
- https://addons.mozilla.org/en-US/firefox/addon/violentmonkey/versions/

**Comment by tophf** [comment, marked as answer]
They take a long time to review. That's how AMO works.

**Reply by hellodword** [reply]
Oh, I see, thanks. I just compared v2.20.0 and v2.19.0, and they were both released on the same day, so I assumed that was the normal pace.

**Reply by tophf** [reply]
Same day?
2.19: May 18
2.20: Jul 8

**Reply by hellodword** [reply]
Sorry, I didn't express that accurately. I mean the release dates on AMO and GitHub.

https://addons.mozilla.org/en-US/firefox/addon/violentmonkey/versions/

```
Version 2.20.0
Released Jul 8, 2024 - 598.36 KB

Version 2.19.0
Released May 18, 2024 - 595.1 KB
```

**Reply by tophf** [reply]
That's because the date of an existing release was automatically updated after AMO signed the package and we attached the signed xpi to the release.

**Reply by hellodword** [reply]
That makes sense! Thanks.
